src/initialization.js

```diff
@@ -27,9 +29,30 @@ var initialization = {
                 isInitialized = true;
                 loadEventsAndPages();
             }
+
+            if (!window.optimizelyClientInstance) {
```

Comment:
This assumes we are always loading the full stack sdk. The way we check for a full stack SDK is by seeing if they filled out `sdkKey`, so:

```suggestion
            if (settings.sdkKey && !window.optimizelyClientInstance) {
```